release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1283,7 +1283,7 @@ In addition to the OCI image, the image-based upgrade uses the `ostree` library
 For more information, see xref:../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#understanding-image-based-upgrade-for-sno[Understanding the image-based upgrade for {sno} clusters].
 
 [id="ocp-4-16-edge-computing-ipsec-encryption-for-managed-clusters-ztp_{context}"]
-==== Deploying IPsec encryption to managed clusters with {ztp} and {rh-rhacm} (Technology Preview)
+==== Deploying IPsec encryption to managed clusters with {ztp} and {rh-rhacm}
```

Comment:
This /lgtm. I _think_ this is going to need CM and I wonder if this will need added to the TP/GA networking table 18 in this doc.

Reply:
A table update would also be needed:

![Screenshot from 2024-08-19 15-12-27](https://github.com/user-attachments/assets/75926808-5abe-4d86-b8fa-d13430b984f4)

Reply:
Updated as per the suggestion @dfitzmau